tests-runner/Dockerfile

```diff
@@ -22,13 +22,13 @@ SHELL ["/bin/bash", "-c"]
 # rust stuff
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs -o rustup-init.sh
 RUN chmod +x rustup-init.sh
-RUN ./rustup-init.sh -y -c rustfmt -c clippy --default-toolchain nightly-2024-10-11
+RUN ./rustup-init.sh -y -c rustfmt -c clippy --default-toolchain nightly-2024-04-15
 ENV PATH="/root/.cargo/bin:$PATH"
-RUN rustup target add --toolchain nightly-2024-10-11 aarch64-unknown-linux-gnu x86_64-unknown-linux-gnu x86_64-apple-darwin aarch64-apple-darwin
+RUN rustup target add --toolchain nightly-2024-04-15 aarch64-unknown-linux-gnu x86_64-unknown-linux-gnu x86_64-apple-darwin aarch64-apple-darwin
```

Comment:
Did you break something with the previous PR?

Reply:
good catch ;(